### PR TITLE
Add quality requirement for creating OpenShift projects

### PR DESCRIPTION
**Pull request by simu**
Will be linked in https://github.com/appuio/component-appuio-cloud/pull/63

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Response::
+Project created
+
+Response measure::
+Project (and associated namespace) belongs to the user's default organization
+
+Rationale::
+To enable an OpenShift native experience.
+Developers should be able to create projects in the OpenShift web console or with `oc new-project bar` as if they're on their own OpenShift cluster.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc:23]:
```suggestion
Namespace ownership must not be tied to individuals but to the organization.
This prevents that team mates get locked out from a namespace should its creator forgett to update its permission.
We consider, giving all members access by default, a safer default.
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]
The rational does not reflect our actual intent. Feel free to refine my suggestion.

File: docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc (hunk 1 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc:23]: We need to cover both. We can do them separately or mixed. I am uncertain which is better, but tend towards separate requirements. So I guess this one can be merged as is.

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc (hunk 1 above)
Comment (simu) [reply docs/modules/ROOT/pages/references/quality-requirements/usability/create-ns-with-oc-new-project.adoc:23]: I'll add another page for the organization-level RBAC requirement.

**Pull request merged**